pandasai/__init__.py

```diff
@@ -176,9 +176,8 @@ def run_code(
             return eval(last_line[6:-1])
         # Evaluate last line and return its value or the captured output
         try:
-            result = eval(last_line)
-            return result
-        except:
+            return eval(last_line)
+        except Exception:
```

Comment:
The linter complains about using a generic exception!

Reply:
Since here should be a generic exception, I'll add a pylint disable.